R/recode-with-table.R

```diff
@@ -268,6 +268,58 @@ rec_with_table <-
       database_name <- deparse(substitute(data))
     }
 
+    # The next code chunk will create the variable details file that does not
+    # have any custom variables. All variables which implement a custom
+    # variable will have their specifications replaced by those of the custom
+    # variable.
+    # This is the variable details sheet that should be used for the
+    # rest of the program.
+    no_custom_variables_variable_details <- variable_details
+    # If the custom variable isn't in the variable details sheet then this
+    # is a pre custom variable sheet. Don't run the code.
+    if(variable_details_columns$custom_variable$name %in% colnames(variable_details)) {
```

Comment:
@yulric `variable_details_columns` in which line is this variable defined?

Reply:
It comes from these lines in [strings.R](https://github.com/Big-Life-Lab/recodeflow/blob/custom-variables-imp/R/strings.R#L109-L117).

You probably have more insight into this but what's the best way to use variables from other files? Do you just source that file or call it from  the package namespace, so `recodeflow:::variable_details_columns`?